dandiapi/api/migrations/0011_asset_access_metadata.py

```diff
@@ -0,0 +1,74 @@
+# Generated by Django 4.1.13 on 2024-08-20 17:21
+from __future__ import annotations
+
+from django.db import migrations, models
+from django.db.models.expressions import RawSQL
+
+
+def remove_access_fields(apps, _):
+    Asset = apps.get_model('api.Asset')
+
+    # Use the postgres jsonb '-' operator to delete the 'access' field from metadata
+    Asset.objects.filter(metadata__access__isnull=False).update(
+        metadata=RawSQL("metadata - 'access'", [])
+    )
```

Comment:
Just a note, I tested this locally and it works as expected. Here is the generated SQL:

```sql
UPDATE "api_asset"
   SET "metadata" = (metadata - 'access')
 WHERE ("api_asset"."metadata" -> 'access') IS NOT NULL
```

Reply:
Why not use Django ORM style for this? i.e., replace `Asset.metadata` with the `access`-less copy, followed by `.save()`?

Reply:
We could do that, but that means we'd be doing as many `.save` calls are there are assets, which seems unnecessarily burdensome, as well as the potential for a race condition (if someone were to update the asset at the same time, either their update may be lost, or this one could be). It would also probably take much longer.